apps/communications/contacts/locales/matcher/matcher.en-US.properties

```diff
@@ -9,17 +9,17 @@ curtainCancel                 = Cancel
 merge                         = Merge
 close                         = Ignore
 
-mergeDuplicateTitle           = Merge duplicates
-suggestedDuplicateContacts    = Suggested duplicate contacts for {{name}}
-mergeActionButtonLabel        = {[ plural(n) ]}
-mergeActionButtonLabel[zero]  = No contacts selected
-mergeActionButtonLabel[one]   = Merge with {{n}} contact
-mergeActionButtonLabel[two]   = Merge with {{n}} contacts
-mergeActionButtonLabel[few]   = Merge with {{n}} contacts
-mergeActionButtonLabel[many]  = Merge with {{n}} contacts
-mergeActionButtonLabel[other] = Merge with {{n}} contacts
+mergeDuplicateTitle                  = Merge duplicates
+suggestedDuplicatesMessage.innerHTML = Suggested duplicate contacts for <bdi>{{name}}</bdi>
```

Comment:
I do agree first option is the best, saw the message in dev-gaia that they landed and had to backout, and agree that won't make it to 2.2

So onece is stable we can get ride of the .innerHTML